package.json

```diff
@@ -3,14 +3,15 @@
   "version": "1.0.0",
   "scripts": {
     "test": "node tools/vuln_valid",
-    "report": "node tools/reporter"
+    "report": "node tools/reporter && jest"
```

Comment:
This should be in test not report.

Reply:
I have updated this to have the tests done by `npm test`

Reply:
@waveywaves:
Actually `test` here means that we test the validity of the data where as the `jest` related test is testing the actual code and they shouldn't be mixed together.

Perhaps we should have:
- test should be jest
- let's add another entry called `verify:reports` and use the vuln_valid for that